addons/rook/template/base/cluster/patches/tmpl-cluster.yaml

```diff
@@ -7,6 +7,8 @@ metadata:
 spec:
   mon:
     count: 1
+  mgr:
```

Comment:
No because we don't ever re-deploy the cluster

https://github.com/replicatedhq/kURL/blob/dbbe7fd7ef6234f153ff8a1ae9ea865369f0734c/addons/rook/template/base/install.sh#L199-L205